feature 1592 moving classes to physical

nelekoehler opened the pull request:
## Workflow checklist

### Automation
Closes #

### PR-Assignee
- [ ] 🐙 Follow the [Pull Request Workflow](https://github.com/OpenEnergyPlatform/ontology/wiki/Pull-request-workflow)
- [ ] 📝 Update the [CHANGELOG.md](https://github.com/OpenEnergyPlatform/ontology/blob/dev/CHANGELOG.md)
- [ ] 📙 Add #'s to `term tracker item`

### Reviewer
- [ ] 🐙 Follow the [Reviewer Guide](https://github.com/OpenEnergyPlatform/ontology/wiki/Pull-request-workflow#reviewer-guide-check-changes-introduced-by-a-pull-request)
- [ ] 🐙 Provided feedback and show sufficient appreciation for the work done

nelekoehler commented:
moving liquid and gaseous worked now, I found my mistake from yesterday, but something went wrong while moving vehicle-kilometre, but I don't see what the mistake is.

l-emele commented on src/ontology/edits/oeo-physical.omn:
```diff
@@ -11867,6 +11867,24 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1486",
+        rdfs:label "vehicle-kilometre"
+    
+    SubClassOf: 
+        OEO_00320001
```
This line probably causes the error. OEO_00320001 is used here, but not declared. If you add a line 
```
Class: OEO_00320001
```
it should work

stap-m commented:
@nelekoehler you didn't remove the doubles annotations from oeo-shared-axioms. I updated the workflow description in #1592 a bit. Maybe that helps you with the whole process.
For reviewing, it is helpful to finish one class movement and commit, before editing the next one.

stap-m commented on src/ontology/edits/oeo-physical.omn:
```diff
@@ -12501,13 +12564,155 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1486",
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1592
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1652"@en,
+        rdfs:label "CO2 emission"@en,
+        rdfs:label "carbon dioxide emission"@en
```
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000118> "carbon dioxide emission"@en
```
I'd like to suggest to change the second label to `alternative label` here.

stap-m commented on src/ontology/edits/oeo-shared-axioms.omn:
```diff
@@ -4028,9 +4028,7 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1485",
     SubClassOf: 
-        <http://purl.obolibrary.org/obo/BFO_0000017>,
-        OEO_00010121 some 
-            (OEO_00030022 or <http://purl.obolibrary.org/obo/BFO_0000030>)
```
Is this axiom deleted on purpose? It is part of `demand`, not `energy service demand`

nelekoehler replied:
<img width="815" alt="Bildschirmfoto 2023-09-14 um 14 07 06" src="https://github.com/OpenEnergyPlatform/ontology/assets/135724657/c1b8c993-e7c0-4683-af19-0ff7c73c003a">

Ah right, that probably happened when I removed the double axioms of energy service demand. I also removed the Subclass of the Ancestor demand, because it was there twice, but I recognised now, that the second one belongs to oeo-shared and should be moved in the according pull request for the shared-classes.

nelekoehler commented:
If the changes here in this PR are fine, I would merge it. For moving the other classes of physical, I will start a new pull request.

stap-m commented:
Looks good for me.

l-emele commented on src/ontology/edits/oeo-shared-axioms.omn:
```diff
@@ -3997,135 +3996,33 @@ https://github.com/OpenEnergyPlatform/ontology/pull/1463"@en,
-        rdfs:label "ton-kilometre"
-    
     SubClassOf: 
         OEO_00320001
```
This axiom (`ton-kilometre SubClassOf: 'transport performance unit'`) appears twice and should be removed from oeo-shared-axioms

l-emele commented on src/ontology/edits/oeo-shared-axioms.omn:
```diff
@@ -3997,135 +3996,33 @@ https://github.com/OpenEnergyPlatform/ontology/pull/1463"@en,
-    
     SubClassOf: 
         <http://purl.obolibrary.org/obo/BFO_0000020>,
         OEO_00010121 some OEO_00010023
```
These SubClassOf axioms are duplicates and should be removed from oeo-shared-axioms

l-emele reviewed (changes requested):
Apart from the ones I commented directly on specific code lines, please check again the following four classes/individuals:
* `vehicle-kilometre`
* `demand`
* `gaseous`
* `liquid`

These still have duplicated annotations and/or axioms.

nelekoehler commented:
> Apart from the ones I commented directly on specific code lines, please check again the following four classes/individuals:
> 
> * `vehicle-kilometre`
> * `demand`
> * `gaseous`
> * `liquid`
> 
> These still have duplicated annotations and/or axioms.

I checked and changed these, except of demand, because that's part of oeo-shared, so it's maybe better to remove the doubles in the according PR. But I also looked again into energy service demand. I hope, I removed every duplicate now.
